resources/user-agents/apps/kmail2-generic.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I think it would be nice to move these to their own folder: `resources/user-agents/apps/kmail2/`